File: documentation-examples-kotlin/src/kotlin/dev/typr/foundations/docs/sqlserver/RowversionType.kt
package dev.typr.foundations.docs.sqlserver

import dev.typr.foundationskt.*
import dev.typr.foundationskt.data.*

@Suppress("unused")
class RowversionType {
    //start
    val rowversionType: SqlServerType<ByteArray> = SqlServerTypes.rowversion
    //stop
}

File: documentation-examples-scala/src/scala/dev/typr/foundations/docs/sqlserver/RowversionType.scala
package dev.typr.foundations.docs.sqlserver
import dev.typr.scalafoundations.*
import dev.typr.scalafoundations.data.*



@SuppressWarnings(Array("unused"))
object RowversionType:
  //start
  val rowversionType: SqlServerType[Array[Byte]] = SqlServerTypes.rowversion
  //stop
